Test results (for internal use)

hero101 opened the pull request.

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20229158390

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20229158390

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/alkem-io/test-suites?pullRequest=482) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/alkem-io/test-suites?pullRequest=482) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: hero101<br/>:x: github-actions[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/alkem-io/test-suites?pullRequest=482) it.</sub>

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20230588158

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20230824100

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20231005323

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20231135706

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20233357064

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20233944710

hero101 committed:
Add nightly reports and server API index page

- Created runinfo.txt and status.txt for nightly report 20233944710 indicating failure.
- Added index.html for nightly Playwright E2E reports with links to test results.
- Introduced index.html for server API with a placeholder message and navigation link.

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20234840384

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20235124334

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20235264679

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20235772207

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20236058276

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20236335725

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20236603212

github-actions[bot] committed:
Update Playwright report: 2025-12-15/20236983719

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20260714920

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20264661559

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20265337016

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20267256623

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20267908880

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20268741443

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20269535673

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20270975509

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20271337079

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20272223776

hero101 committed:
deled old artifacts

github-actions[bot] committed:
Update Playwright report: 2025-12-16/20274014120

github-actions[bot] committed:
Update Playwright report: 2025-12-17/20305610750

github-actions[bot] committed:
Update Playwright report: 2025-12-18/20341288157

github-actions[bot] committed:
Update Playwright report: 2025-12-18/20342049096

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20366740147

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20367550596

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20368684583

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20372215872

hero101 committed:
deleted old data

hero101 committed:
deleted unused actions

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20375201155

github-actions[bot] committed:
Update Playwright report: 2025-12-19/20376027897

github-actions[bot] committed:
Update Playwright report: 2025-12-23/20460854669

github-actions[bot] committed:
Update Playwright report: 2025-12-24/20483448568

github-actions[bot] committed:
Update Playwright report: 2026-01-05/20719931185

github-actions[bot] committed:
Update Playwright report: 2026-01-05/20726670848

Comoque1 commented:
Closing the PR, but the branch must remain as it contains the data for results reporting from the build.
